LICENSE

```diff
@@ -1,8 +1,9 @@
 Fastly Insights.js
-Build generated: <%= moment().format('YYYY-MM-DD') %>
+Build: [hash]
+Generated: [date]
 https://github.com/fastly/insights.js
 
-Copyright (c) <%= moment().format('YYYY') %>, Fastly, Inc. All rights reserved.
+Copyright (c) [year], Fastly, Inc. All rights reserved.
```

Comment:
Cool, didn't know this bracket syntax, much cleaner.

Reply:
Some of them are provided by webpack banner plugin for free, such as `[hash]`. So I just replicated that syntax to add the year and date.